### DISCUSSION TITLE
Internal GUI appears outside WSL

### DISCUSSION THREAD

**Discussion by HengyueLi** [microsoft/WSL#9464] [General]
### Version

Microsoft Windows [Version 10.0.19045.2364]

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Kernel Version

5.15.79.1

### Distro Version

Ubuntu 18.04

### Repro Steps

I use WSL in this way: Inside Ubuntu, I installed XRDP (RDP server). In Windows, I can log in with RDP. Today I updated WSL to the newest version. Then the problem appears. Inside Ubuntu, when I click "terminal". A terminal is launched "outside" of Ubuntu. It is an independent window in Windows. The same situation is happening for other Apps. I guess this should be a new feature. (Maybe the tag "bug" is not appropriate)

### Expected Behavior

All Apps and GUI surfaces should be inside the sub-system, not in Windows directly.

### Actual Behavior

GUI should be inside the subsystem and invisible in Windows.

**Comment by elsaco** [comment]
You didn't post WSL version you upgraded from but beginning with version 1.0.0 WSLg is available on Windows 10 also. It's a feature not a bug!
